profiler/build.gradle

```diff
@@ -6,6 +6,11 @@ apply from: "$rootDir/gradle/shadow.gradle"
 
 def relocatePackages = ext.relocatePackages
 
+compileJava {
+    sourceCompatibility = 8
+    targetCompatibility = 8
```

Comment:
I know that we use `options.release.set(8)` everywhere else, but I could not get gradle to respect the jfr APIs in java 8 with that.  This seems to work, though.